x-pack/test/apm_api_integration/tests/throughput/service_maps.spec.ts

```diff
@@ -113,7 +121,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             [
               ...Object.values(throughputTransactionValues),
               ...Object.values(throughputMetricValues),
-            ].forEach((value) => expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE)));
+            ].forEach((value) =>
+              expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE + GO_PROD_RATE))
+            );
+          });
+        });
+
+        describe('when calling service maps API', () => {
+          let serviceMapsNodeThroughput: number | null | undefined;
+          before(async () => {
+            const response = await callApi();
+            serviceMapsNodeThroughput =
+              response.body.currentPeriod.transactionStats?.throughput?.value;
+          });
+
+          it('returns expected throughput value', () => {
+            expect(roundNumber(serviceMapsNodeThroughput)).to.be.equal(
```

Comment:
The `roundNumber` is needed
<img width="623" alt="image" src="https://user-images.githubusercontent.com/31922082/163787555-9b40a251-57bb-4e60-8a07-2f197a8faa24.png">

Reply:
The second `roundNumber` should not be needed:

<img width="74" alt="CleanShot 2022-04-18 at 12 15 11@2x" src="https://user-images.githubusercontent.com/352732/163794234-636d702c-0dbf-405f-90aa-3291f3d44a3e.png">

Reply:
`roundNumber` is rounding the first value to 20.00, if we don't use r`oundNumber` on the second value the test fail, because 20.00 is not equal to 20

Reply:
It's because `roundNumber` returns a string. I don't think it should, but maybe I am missing something. Can you see what happens if you convert the result of `toPrecision()` in the `roundNumber` function to a number?